WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -604,3 +607,43 @@ private extension PersonViewController {
         }
     }
 }
+
+// MARK: - Jetpack powered badge
+extension PersonViewController {
+
+    private static let jetpackBadgeHeight: CGFloat = 96
```

Comment:
This height value was taken from the badge's height when rendering in App Settings.

Reply:
This results in the actual badge's height being 36, which is consistent with other places in the app 👌